spec/models/manageiq/providers/kubernetes/container_manager_spec.rb

```diff
@@ -47,4 +47,18 @@
       expect(job.zone).to eq("default")
     end
   end
+
+  context "Kubeclient" do
+    it "receives expected arguments" do
+      require 'kubeclient'
+      expect(
+        Kubeclient::Client.new(
+          URI::HTTPS.build(:host => 'hostname', :port => 345),
+          'v1',
+          :ssl_options    => {:verify_ssl => OpenSSL::SSL::VERIFY_NONE},
+          :auth_options   => {:bearer_token => "1234"},
+          :http_proxy_uri => VMDB::Util.http_proxy_uri.to_s
+        )).to be_truthy
```

Comment:
I couldn't make this work with ```reveive(...).with(...)``` probably because of this rspec problem that should be solved here https://github.com/rspec/rspec-expectations/issues/846